pccommon/pccommon/tracing.py

```diff
@@ -24,7 +24,7 @@
             f"InstrumentationKey={_config.app_insights_instrumentation_key}"
         )
     )
-    if _config.app_insights_instrumentation_key is not None
+    if _config.app_insights_instrumentation_key
```

Comment:
Required because using `APP_INSIGHTS_INSTRUMENTATION_KEY=${APP_INSIGHTS_INSTRUMENTATION_KEY}` can result in blank strings, rather than the `- APP_INSIGHTS_INSTRUMENTATION_KEY` that was in the docker-compose environment which will leave it unset if the host environment variable is not set.